README.md

```diff
@@ -57,6 +62,8 @@ See [Basic example](examples/basic/README.md) for further information.
 | ebs\_size | Size of the EBS volume | `string` | `"20"` | no |
 | ebs\_type | Type of the EBS volume | `string` | `"gp2"` | no |
 | instance\_type | Instance type of the instances | `string` | `"t3.medium"` | no |
+| lambda\_function\_zip\_base\_url | Base URL of zip file with lambda function code | `string` | `"https://github.com/lablabs/aws-sf-lambda/releases/download/0.1.1/"` | no |
```

Comment:
Pride mi ze mono by bolo fajn vytiahnut tu verziu von teda aby to malo este aj version string 0.1.1 ako samostatnu variable.